docs/src/swagger-output.json

```diff
@@ -2894,26 +3183,104 @@
           "200": {
             "description": "successful operation",
             "schema": {
-              "$ref": "#/definitions/Email"
+              "type": "array",
+              "items": {
+                "properties": {
+                  "type": {
+                    "type": "string"
+                  },
+                  "required": {
+                    "type": "boolean"
+                  },
+                  "ref": {
+                    "type": "string"
+                  },
+                  "index": {
+                    "type": "boolean"
+                  }
+                }
+              }
             }
+          },
+          "400": {
+            "description": "Invalid Status"
+          }
+        }
+      },
+      "delete": {
+        "tags": [
+          "hexathons"
+        ],
+        "summary": "Deletes a tag by ID.",
+        "description": "",
+        "produces": [
+          "application/json"
+        ],
+        "parameters": [
+          {
+            "name": "id",
+            "in": "path",
+            "type": "integer",
+            "description": "Tag ID",
+            "required": true
+          }
+        ],
+        "responses": {
+          "204": {
+            "description": "Successfully deleted"
+          },
+          "400": {
+            "description": "Invalid Status"
           }
         }
       }
     },
-    "/checkin": {
+    "/sponsor-visit": {
       "get": {
-        "tags": ["hexathons"],
-        "summary": "Gets the checkin information of every user.",
+        "tags": [
+          "hexathons"
+        ],
+        "summary": "Read the sponsor visits.",
         "description": "",
-        "consumes": ["application/json"],
-        "produces": ["application/json"],
+        "consumes": [
+          "application/json"
+        ],
+        "produces": [
+          "application/json"
+        ],
         "responses": {
           "200": {
             "description": "successful operation",
             "schema": {
               "type": "array",
               "items": {
-                "$ref": "#/definitions/Checkin"
+                "properties": {
+                  "visitorId": {
+                    "type": "string"
+                  },
+                  "hexathon": {
+                    "type": "string"
+                  },
+                  "company": {
+                    "type": "string"
+                  },
+                  "employee": {
+                    "type": "string"
+                  },
+                  "starred": {
+                    "type": "boolean"
+                  },
+                  "tags": {
+                    "type": "string"
+                  },
+                  "notes": {
+                    "type": "string"
+                  },
+                  "time": {
+                    "type": "string",
+                    "format": "date-time"
+                  }
+                }
```

Comment:
Instead of repeating this code in multiple areas, we could have this as a separate definition named like `SponsorVisit`, and then just reference it here. Look at the bottom of the docs for examples. This will make it easier for us when adding more endpoints.